### ISSUE TITLE
Can't add table

### ISSUE THREAD

**Issue by CyrilCharlier** [ondras/wwwsqldesigner#268]
Hi,

i just unzip the master repository and i can't add table : 

I use the menu "add table" and it become [place table]

![image](https://user-images.githubusercontent.com/11455745/39957121-982c252a-55ed-11e8-80f9-bf654d38aa10.png)

i click / double click, right and left but nothing is happen.

Any idea ? 

Thanks

**Comment by ondras** [comment]
@CyrilCharlier does this behavior also happen with the hosted version at http://ondras.zarovi.cz/sql/demo/ ?

And what Chrome version? And what OS?

**Comment by CyrilCharlier** [comment]
No not happen on this repo... hmm this is a clone from this repo ?

**Comment by ondras** [comment]
Yes, the hosted version uses the same source code as this repository.
